src/detail-panels/account-header/account-extra-info.jsx

```diff
@@ -110,6 +110,8 @@ function MultilineFormatted({ text, lineClassName = 'text-multi-line' }) {
   const lines = textWithSpaces.split('\n');
   const lineElements = [];
   const urlRegex = /(https?:\/\/[^\s]+)/g;
+  const atRegex = /(^|\s)@([\w.]+)/g;
```

Comment:
this is probably not entirely accurate for all possible handles, but close enough for now